packages/core/src/codewhisperer/service/transformByQHandler.ts

```diff
@@ -794,7 +802,6 @@ export async function getTransformationPlan(jobId: string) {
 
 export async function getTransformationSteps(jobId: string) {
     try {
-        await sleep(2000) // prevent ThrottlingException
```

Comment:
When we poll, we do so every 5 seconds, so this was unnecessary.

Reply:
Are you sure, I remember we added in an extra delay at one point when we transitioned to the next step (i.e. prepare -> planned or something like that)

Don't know if this is that one but just ensure that there is no way that another API is called prior to this within 1 second :)

Reply:
Ok, in that case, `GetTransformationJob` may be running at same time, so adding back this 2s sleep for safety.